assets/src/widget/stories-widget.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function ($) {
```

Comment:
Since we use webpack, is this IIFE really needed? Our code is never in global scope.

So for the two places where we need it, we can just do:

```js
const $ = window.jQuery;
///
$(document).on('widget-updated', widgetChange);
$(document).on('widget-added', widgetChange);
```

thoughts?

Reply:
Removed IIFE in favoru of webpack.